Client/Frontend/Library/HistoryPanel/HistoryPanel.swift

```diff
@@ -485,6 +485,12 @@ class HistoryPanel: UIViewController,
     }
 
     func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        if let item = diffableDatasource?.itemIdentifier(for: indexPath),
```

Comment:
let me double check because I think is possible to have 2 actionable items and we should remove the swipe for both